packages/wdio-cucumber-framework/src/utils.js

```diff
@@ -160,3 +168,99 @@ export function getStepFromFeature(feature, pickle, stepIndex, sourceLocation) {
 
     return targetStep
 }
+
+/**
+ * @param {object[]} result cucumber global result object
+ */
+export function getDataFromResult (result) {
```

Comment:
I think the following would be a lot cleaner
```
export const getDataFromResult ([ uri, feature, ...scenarios ]) => ({ uri, feature, scenarios })
```

Reply:
result object has no such properties